x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/multi_page_layout/components/add_first_integration_splash.tsx

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import styled from 'styled-components';
+import useWindowSize from 'react-use/lib/useWindowSize';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import type { EuiImageProps } from '@elastic/eui';
+import {
+  EuiImage,
+  EuiTitle,
+  EuiBottomBar,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiStepNumber,
+  EuiText,
+  EuiSpacer,
+  EuiLink,
+  EuiButtonEmpty,
+  EuiHideFor,
+  EuiShowFor,
+  isWithinMaxBreakpoint,
+} from '@elastic/eui';
+
+import type { RegistryPolicyTemplate, PackageInfo } from '../../../../../types';
+import { IntegrationBreadcrumb } from '../../components';
+import { Error } from '../../../../../components';
+import { pkgKeyFromPackageInfo } from '../../../../../services';
+import { WithHeaderLayout } from '../../../../../layouts';
+import { useStartServices } from '../../../../../hooks';
+import type { RequestError } from '../../../../../hooks';
+
+const PaddedCentralTitle = styled('h1')`
```

Comment:
I've had to use more custom CSS than I'd like here to get responsive layout working, suggestions welcome!